backend/src/schema/types/type/NOTIFIED.gql

```diff
@@ -0,0 +1,28 @@
+type NOTIFIED {
+  from: NotificationSource
+  to: User
+  createdAt: String
+  read: Boolean
+  reason: NotificationReason
+}
+
+union NotificationSource = Post | Comment
+
+enum NotificationOrdering {
+  createdAt_asc
+  createdAt_desc
+}
+
+enum NotificationReason {
+  mentioned_in_post
+  mentioned_in_comment
+  comment_on_post
```

Comment:
I know that I did wrote this, but shouldn’t that be `commented_on_post`?
To have the database plausible. This we can not just change very easy after the release …

(This change would have a little effect on the frontend as well …)